content/api/graphs/graphs_snapshot.md

```diff
@@ -6,6 +6,9 @@ external_redirect: /api/#graph-snapshot
 ---
 
 ## Graph snapshot
+
+**Note**: When a snapshot is created, [there is some delay](http://andreafalzetti.github.io/blog/2017/04/17/datadog-png-snapshot-not-showing.html) before it is available.
```

Comment:
Should we link an external personal blog to the docs? Maybe a FAQ would be more suited.

I'm just thinking about what would happen when the person deletes their blog article.

Reply:
you are right, but I found this blog article pretty well written, hence the link to it.
If this page disappears, one of our CI tests will spot it and we will just remove the href. (no link no problem 💯 )